tutorials/ChannelMiddleware.onError.md

```diff
@@ -0,0 +1,42 @@
+Register an error handler. This is called before responding on any error.
+
+### Channel Provider
+```js
+(async ()=> {
+    const provider = await fin.InterApplicationBus.Channel.create('channelName');
+
+    provider.register('provider-action', (payload, identity) => {
+        console.log(payload);
+        throw new Error('Action error');
+        return {
```

Comment:
unreachable code

Reply:
that's fine, want to make sure returning is a part of every `registry` call.

Reply:
Could we make the throw conditional?

Reply:
why? 
accomplishes what it set out to do.